web-server/src/components/Service/SystemStatus.tsx

```diff
@@ -0,0 +1,174 @@
+  [ServiceNames.SYNC_SERVER]: '#ab34eb'
+};
+
+export const SystemStatus: FC = () => {
+  const dispatch = useDispatch();
+  const theme = useTheme();
+  const loading = useSelector((state) => state.service.loading);
+  const services = useSelector((state) => state.service.services);
+
+  useEffect(() => {
+    const eventSource = new EventSource(`/api/stream`);
+
+    eventSource.onmessage = (event) => {
+      const data = JSON.parse(event.data);
+
+      if (data.type === StreamEventType.StatusUpdate) {
+        dispatch(serviceSlice.actions.setStatus({ statuses: data.statuses }));
+      }
+      if (data.type === StreamEventType.LogUpdate) {
+        const { serviceName, content } = data;
+        const newLines = content.split('\n');
+        const trimmedLines = newLines.filter(
+          (line: string) => line.trim() !== ''
+        );
+        dispatch(
+          serviceSlice.actions.setServiceLogs({
+            serviceName,
+            serviceLog: trimmedLines
+          })
+        );
+      }
+    };
+
+    return () => {
+      eventSource.close();
+      dispatch(serviceSlice.actions.resetState());
+    };
+  }, [dispatch]);
+
+  const { addPage } = useOverlayPage();
+
+  const handleCardClick = (serviceName: ServiceNames) => {
+    addPage({
+      page: {
+        ui: 'system_logs',
+        title: `${serviceTitle[serviceName]} Logs`,
+        props: { serviceName }
+      }
+    });
+  };
+
+  return (
+    <FlexBox col gap={2} sx={{ padding: '16px' }}>
+      <Line bold white fontSize="24px" sx={{ mb: 2 }}>
+        System Status
+      </Line>
+      <Divider sx={{ mb: 2, backgroundColor: 'rgba(255, 255, 255, 0.2)' }} />
+
+      {loading ? (
+        <Box
+          sx={{
+            display: 'flex',
+            alignItems: 'center',
+            justifyContent: 'center'
+          }}
+        >
+          <CircularProgress color="primary" size={50} />
+        </Box>
+      ) : (
+        <FlexBox col gap={2}>
+          {Object.keys(services).map((serviceName) => {
+            const serviceKey = serviceName as ServiceNames;
+            const { isUp } = services[serviceKey];
+            const borderColor = serviceColor[serviceKey];
+
+            return (
+              <CardRoot
+                key={serviceName}
+                onClick={() => handleCardClick(serviceKey)}
+                sx={{
+                  transition: 'box-shadow 0.2s ease',
+                  '&:hover': {
+                    boxShadow: '0 4px 15px rgba(0, 0, 0, 0.1)'
+                  },
+                  backgroundColor: 'rgba(255, 255, 255, 0.05)',
+                  borderRadius: '12px',
+                  border: `1px solid ${
+                    isUp ? borderColor : alpha(theme.colors.error.main, 0.3)
+                  }`,
+                  padding: '16px',
+                  cursor: 'pointer',
+                  position: 'relative',
+                  overflow: 'hidden'
+                }}
+              >
```

Comment:
rather then using `CardRoot` use `Button`, it will work same way with most styling issues sorted for you.